<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NBioBSP_Demo</class>
 <widget class="QDialog" name="NBioBSP_Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>759</width>
    <height>487</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>759</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NBioBSP_Demo</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>533</width>
     <height>477</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="editEIQ">
      <property name="inputMask">
       <string>999; </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Verify Image Quality</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editVIQ">
      <property name="inputMask">
       <string>999; </string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Max Enroll Finger</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Sample per Finger</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Default Timeout</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Security Level</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editMEF">
      <property name="inputMask">
       <string>99; </string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editSPF">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="inputMask">
       <string>999; </string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="editDTO">
      <property name="inputMask">
       <string>999999; </string>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboSL"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Enroll Image Quality</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btSetInit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Set Initial Info</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btGetInit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Get Initial Info</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Device List</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Initialization</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="comboDevice"/>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="btOpen">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Registration</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Payload Value</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="editPayload"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Verify</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>FIR Data Type</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QComboBox" name="comboDataType"/>
    </item>
    <item row="16" column="2">
     <widget class="QPushButton" name="btVerify">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Verify</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="3">
     <widget class="QLabel" name="labelStatus">
      <property name="text">
       <string>Status Text</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="3">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="btCapture">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Capture</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="FPReg">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FPVerify">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editEIQ</tabstop>
  <tabstop>editVIQ</tabstop>
  <tabstop>editMEF</tabstop>
  <tabstop>editSPF</tabstop>
  <tabstop>editDTO</tabstop>
  <tabstop>comboSL</tabstop>
  <tabstop>btGetInit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
